data/scripts/bin/apply-operator-crs.sh.template

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+export KUBECONFIG="/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost.kubeconfig"
+
+ensure_csvs_succeeded() {
+  local namespace="$1"
+
+  # Get all CSV names in the specified namespace
+  local csv_names=$(oc get csv -n "$namespace" -o jsonpath='{.items[*].metadata.name}')
+
+  if [ -z "$csv_names" ]; then
+    echo "No CSVs found yet in namespace '$namespace'."
+    return 1
+  fi
+
+  # Iterate through each CSV name
+  IFS=' ' read -r -a csv_array <<< "$csv_names"
+  for csv in "${csv_array[@]}"; do
+    # Get the status of the current CSV
+    local status=$(oc get csv -n "$namespace" "$csv" -o jsonpath='{.status.phase}')
+    if [ "$status" != "Succeeded" ]; then
+      return 1
+    fi
+  done
+
+  return 0
+}
+
+# Loop over the CRs
+crsDir="/etc/assisted/extra-manifests/crs"
```

Comment:
From a user point of view `crs` it's a little bit vague.  Since it's something specific to OLM operators - but anyhow meant to applied post-installation, I would find more intuitive to use something linke `/etc/assisted/extra-manifests/post-installation` . This will eliminate also the ambiguity of the other manifests in `/etc/assisted/extra-manifest`, that will be applied instead during the installation.